siac/daemoncmd.go

```diff
@@ -27,13 +28,36 @@ var (
 		Long:  "Check for available updates.",
 		Run:   wrap(updatecheckcmd),
 	}
+
+	versionCmd = &cobra.Command{
+		Use:   "version",
+		Short: "Print version information",
+		Long:  "Print version information.",
+		Run:   wrap(versioncmd),
+	}
 )
 
 type updateInfo struct {
 	Available bool   `json:"available"`
 	Version   string `json:"version"`
 }
 
+type daemonVersion struct {
+	Version string
+}
```

Comment:
Also, I don't think it's strictly necessary, but it might be good practice to have the JSON tags for the version field as well.